helm/reana/values.yaml

```diff
@@ -49,6 +49,20 @@ secrets:
   cern:
     sso: {}
   reana: {}
+  keycloak:
+    consumer_key: ""
+    consumer_secret: ""
+
+# Keycloak configuration (also refer to secrets.keycloak.*)
+keycloak_config:
+  enabled: false
+  title: ""
+  desciption: ""
+  base_url: ""
+  auth_url: ""
+  token_url: ""
```

Comment:
I think we should also add the `OAUTHCLIENT_KEYCLOAK_REALM_URL`, as I think that the URL generated by the helper is probably going to be wrong with deployments of keycloak that do not use realms